docker/docker_set_up.sh

```diff
@@ -0,0 +1,3 @@
+rm tmp/pids/server.pid
+yarn install --check-files
+bundle exec rails server -b 0.0.0.0
```

Comment:
@abhinavtripathy any specific reason why we're starting the server on this particular port?

Reply:
it references the port specified in docker-compose